packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -57,7 +57,10 @@ export async function startConsolidatedQuery({
     dob: patient.data.dob,
     genderAtBirth: patient.data.genderAtBirth,
     address: patient.data.address,
+    cxDocumentRequestMetadata: cxConsolidatedRequestMetadata,
     cxConsolidatedRequestMetadata: cxConsolidatedRequestMetadata,
+    whOverride:
+      (cxConsolidatedRequestMetadata as { whOverrideFlag?: string })?.whOverrideFlag !== undefined,
```

Comment:
I suggest we create a simple function for this so:
1. its isolated from the rest of the code;
2. can be reused so we don't write this twice.

Also, it would be good to have the property/type/name defined in one place and referenced throughout, so we don't have the risk of updating it in one place and forgetting to update other places. By using `... as { whatever: string }` we're creating a new type that's disconnected from the one used somewhere else.

Actually, if we have this function inside `cxDocumentRequestMetadata` and `cxConsolidatedRequestMetadata` we should be able to simplify this PR to be:
- One function to check if `disableWH` is enabled
- Two conditionals calling that function

```ts
function isDisableWH(meta?: unknown): boolean {
  if (!meta) return false;
  return Boolean((meta as any).disableWH);
}
```

...used like:

```ts
  // send it to the customer and update the WH request status
  if (!isDisableWH(currentPatient.data.cxConsolidatedRequestMetadata)) {
    await processRequest(...);
  }
```